src/main/java/de/tum/in/www1/artemis/domain/ProgrammingExercise.java

```diff
@@ -836,4 +840,13 @@ public String getBuildPlanAccessSecret() {
     public void generateAndSetBuildPlanAccessSecret() {
         buildPlanAccessSecret = UUID.randomUUID().toString();
     }
+
+    @Nullable
+    public Boolean getIrisActivated() {
```

Comment:
You usually use something like `is` or `has` instead of `get` for booleans in Java.
Also should probably not be nullable.
```suggestion
    public boolean isIrisActivated() {
```